### ISSUE TITLE
Content: Creation of our accessibility statement

### ISSUE THREAD

**Issue by JulieBB23** [gc-da11yn/user-experience-planning#3]
We should have an accessibility statement, here an example [https://accessible.canada.ca/accessibility-statement](url)
